framework/src/main/java/org/checkerframework/framework/qual/CFComment.java

```diff
@@ -0,0 +1,41 @@
+package org.checkerframework.framework.qual;
+
+import static java.lang.annotation.ElementType.*;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * This annotation is to be used to add comments related to Checker-Framework. This annotation can
+ * be used on packages, classes, interfaces and any elements inside these.
```

Comment:
You can just say this is a declaration annotation, or leave it out since it is obvious from the